profiles/kentik_snmp/dell/emc-data-domain.yml

```diff
@@ -0,0 +1,361 @@
+# http://oid-info.com/get/1.3.6.1.4.1.19746.3.1
+---
+extends:
+  - system-mib.yml
+  - if-mib.yml
+
+provider: kentik-nas
+
+sysobjectid:
+  - 1.3.6.1.4.1.19746.3.1.*    # Captures all devices in the Data Domain root
+
+metrics:
+  # Memory Total
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      OID: 1.3.6.1.2.1.25.2.3.1.6.1
+      name: hrStorageUsed
+      tag: MemoryTotal
+  # Memory Buffers
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: hrStorageUsed
+      OID: 1.3.6.1.2.1.25.2.3.1.6.6
+      tag: MemoryBuffer
+  # Memory Cache
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: hrStorageUsed
+      OID: 1.3.6.1.2.1.25.2.3.1.6.7
+      tag: MemoryCache
+  # Status of the file system
+  - MIB: DATA-DOMAIN-MIB
+    symbol:
+      name: fileSystemStatus
+      OID: 1.3.6.1.4.1.19746.1.3.1.1.0
+      enum:
+        enabled: 1
+        disabled: 2
+        running: 3
+        unknown: 4
+        error: 5
+        cleaning: 6
+  # Amount of Uncompressed data in Gigabytes that has been backed up by the system
+  - MIB: DATA-DOMAIN-MIB
+    symbol:
+      name: fileSystemVirtualSpace
+      OID: 1.3.6.1.4.1.19746.1.3.1.2.0
+  # File system uptime
+  - MIB: DATA-DOMAIN-MIB
+    symbol:
+      name: fileSystemUptime
+      OID: 1.3.6.1.4.1.19746.1.3.1.3.0
+  # File system status message
+  - MIB: DATA-DOMAIN-MIB
+    symbol:
+      name: fileSystemStatusMessage
+      OID: 1.3.6.1.4.1.19746.1.3.1.4.0
+  # A table containing entries of PowerModuleEntry
+  - MIB: DATA-DOMAIN-MIB
+    table:
+      name: powerModuleTable
+      OID: 1.3.6.1.4.1.19746.1.1.1.1.1
+    symbols:
+      # Current enclosure Power Module status
+      - name: powerModuleStatus
+        OID: 1.3.6.1.4.1.19746.1.1.1.1.1.1.4
+        enum:
+          absent: 0
+          ok: 1
+          failed: 2
+          faulty: 3
+          acnone: 4
+          unknown: 99
+    metric_tags:
+      # Power Module Enclosure ID
+      - column:
+          name: powerEnclosureID
+          OID: 1.3.6.1.4.1.19746.1.1.1.1.1.1.1
+      # Power Module index
+      - column:
+          name: powerModuleIndex
+          OID: 1.3.6.1.4.1.19746.1.1.1.1.1.1.2
+      # Power Module Description
+      - column:
+          name: powerModuleDescription
+          OID: 1.3.6.1.4.1.19746.1.1.1.1.1.1.3
+  # A table containing entries of TemperatureSensorEntry
+  - MIB: DATA-DOMAIN-MIB
+    table:
+      name: temperatureSensorTable
+      OID: 1.3.6.1.4.1.19746.1.1.2.1.1
+    symbols:
+      # Current Temperature Value of the sensor in Celsius
+      - name: tempSensorCurrentValue
+        OID: 1.3.6.1.4.1.19746.1.1.2.1.1.1.5
+      # Current status of the sensor
+      - name: tempSensorStatus
+        OID: 1.3.6.1.4.1.19746.1.1.2.1.1.1.6
+        enum:
+          failed: 0
+          ok: 1
+          notfound: 2
+          overheatWarning: 3
+          overheatCritical: 4
+    metric_tags:
+      # Temperature Sensor Number
+      - column:
+          name: tempSensorIndex
```

Comment:
Do we still need to specifically collect index columns?  I believe @i3149 had set it up that Index is automatically collected for all tabular oids?

Reply:
Correct, there is an implicit Index which will be set for all tables.